### ISSUE TITLE
plugin v3.0.0 missing Environment Variables in Artifactory tasks

### ISSUE THREAD

**Issue by spring-buildmaster** [jfrog/bamboo-artifactory-plugin#97]
We followed the Atlassian upgrade process from Bamboo 2.8.3 -> 2.10.4 by disabling the plugin, upgrading, then upgrading the plugin. Now, when opening any Artifactory Maven task for configuration, the `Environment Variables` field is empty. It also does not seem to be applied in the builds.
It is still present in the schema, e.g. at taskDefinition config key `builder.artifactoryGradleBuilder.environmentVariables`, however, if we save the task configuration again the key is silently overwritten with an empty value. 

So, basically, all our plan Environment Variables are gone, we can't add them back, and the upgrade is blocked.

possibly related to issue #96 ?

**Comment by barbelity** [comment]
@spring-buildmaster thanks for opening this issue.
We have created a snapshot version of the plugin, containing a fix for the bug.
Can you please install the plugin and check if the problem is resolved?

The snapshot version can be downloaded from here:
https://oss.jfrog.org/oss-snapshot-local/org/jfrog/bamboo/bamboo-artifactory-plugin/3.0.x-SNAPSHOT/bamboo-artifactory-plugin-3.0.x-SNAPSHOT.jar
The relevant commit: https://github.com/jfrog/bamboo-artifactory-plugin/commit/f1cbbe56f8bf754907ce33ffa616c383900b2a44.

Your feedback for it is highly appreciated.

**Comment by spring-buildmaster** [comment]
Thank you for the quick fix. This patch has fixed the issue with the missing Environment variables.
Issue #96 persists, so probably unrelated.
